README.md

```diff
@@ -1,2 +1,81 @@
-# hw01
-Repository for homework 01
+
+# HW 01: A bit about me
+#### Author: Patrick Thelen
+#### Created on: 04/03/2019
+
+***
+
+> "I liked things better when I didn't understand them."  
+>   
+> Bill Watterson
+
+
+#### Here's a photo of my grandfather and I on a hike in Spain last summer (I'm on the left).
+![](spain.png)
```

Comment:
To size the figure, if you use RMarkdown to create md files, one option is changing the figure config below. Also, if you want, you can also put the words you'd like to appear in case the picture source is not working, like alternative sources, or "picture missing" in the [].  

```
---
title: "My Document"
output:
  html_document:
    fig_width: 6
    fig_height: 4
---
```

Reply:
Thank you!! Much appreciated.